.coderabbit.yaml

```
@@ -0,0 +1,41 @@
+# yaml-language-server: $schema=https://coderabbit.ai/integrations/schema.v2.json
+# CodeRabbit Configuration
+# Documentation: https://docs.coderabbit.ai/
+
+# Review settings
+reviews:
+  # Enable automatic code reviews on pull requests
+  auto_review: true
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)